### PR TITLE
Updated validator flags

### PR DESCRIPTION
**Pull request by ml-evs**
Preparing for https://github.com/Materials-Consortia/optimade-python-tools/pull/395 to get merged, this PR updates the flags to match the new versions in the validator.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -29,27 +29,38 @@ if echo ${INPUT_VERBOSITY} | grep -Eq '[^0-9]'; then
 fi
 echo "Using verbosity level: ${INPUT_VERBOSITY}"
 
-i=0
-while [[ $i -lt $INPUT_VERBOSITY ]]
-do
+for (( i=0; i<${INPUT_VERBOSITY}; i++ )); do
```
Comment (ml-evs) [inline entrypoint.sh:32]: Haha, I tried *exactly* this locally and it complained about syntax errors

File: entrypoint.sh [hunk 2]
```diff
@@ -29,27 +29,38 @@ if echo ${INPUT_VERBOSITY} | grep -Eq '[^0-9]'; then
...
+        echo "Non-valid input for 'skip_optional': ${INPUT_SKIP_OPTIONAL}. Will use default (false)."
+        ;;
+esac
 
-if [ -n "${INPUT_FAIL_FAST}" ]; then
-    echo "Will exit on first failure."
-    run_validator="${run_validator} --fail-fast"
-fi
+case ${INPUT_FAIL_FAST} in
+    y | Y | yes | Yes | YES | true | True | TRUE | on | On | ON)
```
Comment (ml-evs) [inline entrypoint.sh:54]:
```suggestion
    y | Y | yes | Yes | YES | true | True | TRUE | on | On | ON | 1)
```

thoughts? Can add 0 for "no" too

File: entrypoint.sh (hunk 1 above)
Comment (CasperWA) [reply entrypoint.sh:32]: Not for me :) Also not if you run tests in the Docker image :) Does it still not work for you? What version of bash do you have?

File: entrypoint.sh (hunk 2 above)
Comment (CasperWA) [reply entrypoint.sh:54]: All these options are the official boolean values for YML files, so I don't think I would use also 1 and 0 unless they are accepted as valid boolean values in YML?

Comment (CasperWA) [reply entrypoint.sh:54]: https://yaml.org/type/bool.html

Comment (ml-evs) [reply entrypoint.sh:54]: Fair enough, forgot that the yaml is the main config here and not env vars

File: entrypoint.sh (hunk 1 above)
Comment (ml-evs) [reply entrypoint.sh:32]:
```
./entrypoint.sh: line 32: ((: i<: syntax error: operand expected (error token is "<")
```
```
$ /usr/bin/env bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
```

Comment (CasperWA) [reply entrypoint.sh:32]: Ah.. huh.. yeah. It seems v5 deprecated the C-style for loop... hmm

Comment (CasperWA) [reply entrypoint.sh:32]: I'll revert to the the more chunky while loop 👍

Comment (CasperWA) [reply entrypoint.sh]: I was running v4.4...

**Comment by CasperWA** [comment]
The latest commit uses `master` branch of `optimade-python-tools` for this action's CI.
This commit should be reverted once a new version of `optimade` has been released.

**Comment by CasperWA** [comment]
merge at will :)

**Pull request merged**